### PR TITLE
Sync REPL and browser loader to version and /loader changes

### PR DESCRIPTION
**Pull request by johnjbarton**
When the TraceurLoader files were moved from `/runtime` to `'/loader` a few browser-oriented uses were missed.

When `System` was removed as a global, a few too many calls were changed to point to `ModuleStore`.
The bootstrap functions need to use `ModuleStore` (via `$traceurRuntime` as our single global entry). The calls within our loader should use `this`, not `ModuleStore`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -102,7 +102,8 @@ test: bin/traceur.js \
 	  test-interpret-absolute test-inline-module-error \
 	  test-version \
 	  test/features \
-	  test-experimental
+	  test-experimental \
+	  bin/BrowserSystem.js
```
Comment (arv) [inline Makefile]: Can you put this before all the `test/`?

File: src/loader/InternalLoader.js [hunk 2]
```diff
@@ -355,7 +355,7 @@ export class InternalLoader {
   }
 
   getOrCreateCodeUnit_(name, referrerName, address, metadata) {
-    var normalizedName = ModuleStore.normalize(name, referrerName, address);
+    var normalizedName = System.normalize(name, referrerName, address);
```
Comment (arv) [inline src/loader/InternalLoader.js]: Not clear why this should be changed? Can you add a comment?

**Comment by arv** [comment]
Care to expand the PR description a bit?

File: Makefile (hunk 1 above)
Comment (johnjbarton) [inline Makefile]: done

File: src/loader/InternalLoader.js (hunk 2 above)
Comment (johnjbarton) [inline src/loader/InternalLoader.js]: I added info to the commit description. The change above worked but it was wrong.  We should be calling `this.loader_.normalize()`.  `System` would give the same result but we should not be going through global.

**Comment by johnjbarton** [comment]
updated the description. PTAL

File: src/loader/InternalLoader.js [hunk 3]
```diff
@@ -376,7 +374,8 @@ export class InternalLoader {
         if (bundledModule) {
           codeUnit = new BundledCodeUnit(this.loaderCompiler, normalizedName,
               name, referrerName, address,
-              bundledModule.deps, bundledModule.execute);
+              bundledModule.deps, bundledModule.execute,
+              (key, module) => this.loader_set(key, module));
```
Comment (arv) [inline src/loader/InternalLoader.js]: `loader_set`? Should this be `loaderSet` or `loader_.set`?

Comment (johnjbarton) [inline src/loader/InternalLoader.js]: Since this code is obviously not being tested, I'll just remove it.

**Comment by johnjbarton** [comment]
PTAL

**Comment by arv** [comment]
LGTM

**Pull request closed without merging**